test/e2e/pivoting_based_feature_test.go

```diff
@@ -52,13 +52,19 @@ var _ = Describe("Testing features in ephemeral or target cluster", func() {
 		certRotation(managementCluster.GetClientSet(), managementCluster.GetClient())
 		nodeReuse(managementCluster.GetClient())
 
-		if !ephemeralTest {
-			rePivoting()
-		}
-
 	})
 
 	AfterEach(func() {
+		if !ephemeralTest {
+			// Dump the target cluster resources before re-pivoting.
+			framework.DumpAllResources(ctx, framework.DumpAllResourcesInput{
```

Comment:
Is dumping before pivoting / after repivoting logs would be also useful?

Reply:
I do not understand what do you mean before pivoting or after repivoting there is not CRS on the target cluster what I should dump !!

Reply:
Of course not in the target cluster, since both would be in the management cluster. Maybe we even dump it after repivoting

Reply:
There no problem when it comes to the other bootstrap cluster it is dumping everything in the function `dumpSpecResourcesAndCleanup`

Reply:
yeah, just wanted to double check it